docs/reuse/reference/extensions/integrations.rst

```diff
@@ -22,206 +22,248 @@ integrations, for internal purposes.
         interface: ingress
         limit: 1
 
-In addition to these integrations, in each ``provides`` and ``requires``
-block you may specify further integration endpoints, to integrate with
+In addition to these relations, in each ``provides`` and ``requires``
+block you may specify further relations endpoints, to integrate with
 the following charms:
 
-- Ingress: `traefik <https://charmhub.io/traefik-k8s>`__ and `nginx
-  ingress integrator <https://charmhub.io/nginx-ingress-integrator>`__
-- MySQL: `machine <https://charmhub.io/mysql>`__ and
-  `k8s <https://charmhub.io/mysql-k8s>`__ charm
-- PostgreSQL: `machine <https://charmhub.io/postgresql>`__ and
-  `k8s <https://charmhub.io/postgresql-k8s>`__ charm
-- `MongoDB <https://charmhub.io/mongodb>`__
-- `Canonical Observability Stack
-  (COS) <https://charmhub.io/cos-lite>`__
-- `Redis <https://charmhub.io/redis-k8s>`__
-- `SAML <https://charmhub.io/saml-integrator>`__
-- `S3 <https://charmhub.io/s3-integrator>`__
-- RabbitMQ: `machine <https://charmhub.io/rabbitmq-server>`__ and
-  `k8s <https://charmhub.io/rabbitmq-k8s>`__ charm
-- `Tempo <https://charmhub.io/topics/charmed-tempo-ha>`__
-- `SMTP <https://charmhub.io/smtp-integrator>`__
-- `OpenFGA <https://charmhub.io/openfga-k8s>`__
-
-These endpoint definitions are as below:
+.. list-table::
+  :widths: 30 30
+  :header-rows: 1
+
+  * - Relation
+    - Endpoint definition
+  * - Ingress: `traefik <https://charmhub.io/traefik-k8s>`__ and `nginx
+      ingress integrator <https://charmhub.io/nginx-ingress-integrator>`__
+    - Already available in the charm
+  * - MySQL: `machine <https://charmhub.io/mysql>`__ and
+      `k8s <https://charmhub.io/mysql-k8s>`__ charm
```

Comment:
For the generic descriptor, not the charm name, should it be "Kubernetes charm"? The reference suggests so. https://documentation.ubuntu.com/juju/3.6/reference/charm/#kubernetes